src/renderer/utils/Directories.ts

```diff
@@ -0,0 +1,17 @@
+import Store from "electron-store";
+import path from "path";
+import os from "os";
+
+const settings = new Store;
+
+export class Directories {
+
+    static inCommunity(targetDir: string): string {
+        return path.join(settings.get('mainSettings.msfsPackagePath') as string, targetDir);
+    }
+
+    static temp(): string {
+        return path.join(os.tmpdir(), 'flybywire_installer');
```

Comment:
```suggestion
        return path.join(settings.get('mainSettings.msfsPackagePath') as string, `flybywire_current_install_${(Math.random() * 1000).toFixed(0)}`);
```

Reply:
Do we wanna always use the Community folder now ?

Reply:
As discussed in Discord it is faster and no files are kept locked by fragmenter which caused an issue when deleting before.